spec/system/jobseekers/jobseekers_can_search_for_schools_spec.rb

```diff
@@ -121,42 +108,20 @@
       visit organisations_path
     end
 
-    it "allows user to filter by special schools" do
```

Comment:
We already filter by special schools in a previous test so I don't think we need to repeat it here.